news/4932.bugfix

```diff
@@ -0,0 +1 @@
+Fixed UnicodeDecodeError on Linux which system encoding is not UTF-8.
```

Comment:
This newsfile is not needed. You can just drop this.

Reply:
Thanks. I will remove it.